### PR TITLE
Replicate test environment to AKS

### PR DESCRIPTION
**Pull request by johnake**
### Context

Replicate test environment to AKS

### Changes proposed in this pull request

Include a summary of the change.

### Guidance to review

Include any useful information needed to review this change.
Include any dependencies that are required for this change.

### Checklist

- [ ] Attach to Trello card
- [ ] Rebased main
- [ ] Cleaned commit history
- [ ] Tested by running locally

### CODE REVIEW

File: .github/workflows/build-and-deploy.yml [hunk 1]
```diff
@@ -4,6 +4,7 @@ on:
   push:
     branches:
       - main
+      - 652_replicate_test_environment_to_aks
```
Comment (RMcVelia) [inline .github/workflows/build-and-deploy.yml:7]: should be removed

File: .github/workflows/build-and-deploy.yml [hunk 2]
```diff
@@ -112,6 +113,31 @@ jobs:
...
+    environment:
+      name: test_aks
+
+    steps:
+      - uses: actions/checkout@v3
+      - uses: ./.github/workflows/actions/deploy_v2
+        id: deploy_v2
+        with:
+          environment: test_aks
+          docker_image: ${{ needs.docker.outputs.docker_image }}
```
Comment (RMcVelia) [inline .github/workflows/build-and-deploy.yml:132]: the docker_image isn't getting through to the k8 app & worker, need to check where this is going wrong. And should remove default 2ad42e8958a4d63ed295a58a0847430705725ba8 from the Makefile

Comment (johnake) [reply .github/workflows/build-and-deploy.yml:132]: Removed the default in Makefile - however I can see the image in K8?

**Comment by johnake** [comment]
[Find a lost TRN - Replicate Test environment to AKS](https://trello.com/c/CPcnKkjw/652-find-a-lost-trn-replicate-test-environment-to-aks)

**Comment by johnake** [comment]
[Find a lost TRN - Replicate Test environment to AKS](https://trello.com/c/CPcnKkjw/652-find-a-lost-trn-replicate-test-environment-to-aks)

**Comment by johnake** [comment]
[Find a lost TRN - Replicate Test environment to AKS](https://trello.com/c/CPcnKkjw/652-find-a-lost-trn-replicate-test-environment-to-aks)

**Review by RMcVelia — APPROVED** [review]

**Pull request merged**